src/components/ProgressBar/index.tsx

```diff
@@ -4,10 +4,16 @@ import ProgressBar from './ProgressBar';
 
 interface WrapperTypes {
   progress: number;
+  progressTitle: string;
 }
 
-const Wrapper: FC<WrapperTypes> = ({ progress }) => {
-  return <ProgressBar progress={progress} />;
+const Wrapper: FC<WrapperTypes> = ({ progress, progressTitle }) => {
```

Comment:
The wrapper (index.tsx) should container interface content (`div`, `p`, etc). Its goal is actually to add logic and hooks that are gonna be passed as props to the interface.

Check out the QuestCard component as example:

`index.tsx`

<img width="757" alt="image" src="https://user-images.githubusercontent.com/43002117/233671384-e9a84214-6fbc-4763-9be3-7ca778efa678.png">

It's where we use the `useNavigate` hook to create the `navigate` variable. Is also where is the logic to decide the Icon we should use and also to declare the `handleClick` function. So is where we use all the hooks, declare variables, functions, etc.

This way, QuestCard.tsx holds no logic, just receives everything as props:

`QuestCard.tsx`

<img width="532" alt="image" src="https://user-images.githubusercontent.com/43002117/233671941-fef11e5b-1919-4dd8-9056-93cdf412b8c8.png">

As a little extra, I'd suggest not adding the <p> to the ProgressBar component, instead leaving just the progress bar itself (so the text would be added on the component importing ProgressBar). If you really want to have the text inside the ProgressBar component since we might not need the isolated bar, I'd still ask you to move the <p> to inside the component.

So in other words:

- Preferable: text outside the ProgressBar component, ProgressBar only exporting the progress bar.

- Acceptable: moving the <p> from the `index.tsx` to the `ProgressBar.tsx`.